tox.ini

```diff
@@ -9,6 +9,8 @@ setenv =
     PYTHONPATH = {toxinidir}
 
 [pytest]
+pep8maxlinelength = 88
+pep8ignore = W503 # W503 is not PEP8 compliant
 addopts=-v -s -ra -l --pep8 --cov=fluentdhec --cov-fail-under 30 --cov-report html
```

Comment:
Can you add mypy to pytest? 
https://pypi.org/project/pytest-mypy/

Reply:
We used the `test.sh` that was placed in the root of the repo for testing with mypy and then tox.

The reason for this is that mypy fails if it is not run inside the `fluentdhec` folder, because of how it follows imports.

I'm looking into using `pytest-mypy`, but for now I cannot replicsate the functionality of `test.sh`. I'll keep looking into it.

Reply:
@denizgenc I think some modification of the `MYPYPATH` might make this possible.

Reply:
I would consider integration of mypy to the test suite a per-requisite for merging.